guides/source/configuring.md

```diff
@@ -432,7 +432,7 @@ The schema dumper adds one additional configuration option:
 
 * `config.action_controller.permit_all_parameters` sets all the parameters for mass assignment to be permitted by default. The default value is `false`.
 
-* `config.action_controller.action_on_unpermitted_parameters` enables logging or raising an exception if parameters that are not explicitly permitted are found. Set to `:log` or `:raise` to enable. The default value is `:log` in development and test environments, and `false` in all other environments.
+* `config.action_controller.action_on_unpermitted_parameters` enables logging or raising an exception if parameters that are not explicitly permitted are found. Set to `:log` or `:raise` to enable. The default value is `:raise` in development and test environments, and `false` in all other environments.
```

Comment:
I believe this change would now affect production's default, too. Perhaps this is a better default production, as well? It seems like intentionally ignoring these params should be an explicit choice in any env.

Reply:
In production we can't raise. It would mean that any wrong parameter sent by users would cause a 500 error. We should just ignore in production.

Reply:
k. That still strikes me as the correct behavior, but I won't die on that hill :)